libpod/container_internal_linux.go

```diff
@@ -1486,6 +1511,24 @@ func (c *Container) restore(ctx context.Context, options ContainerCheckpointOpti
 		}
 	}
 
+	// Restore /dev/shm content
+	if c.config.ShmDir != "" && c.state.BindMounts["/dev/shm"] == c.config.ShmDir {
+		shmDirTarFileFullPath := filepath.Join(c.bundlePath(), metadata.DevShmCheckpointTar)
+		if _, err := os.Stat(shmDirTarFileFullPath); err != nil {
+			logrus.Warn("Container checkpoint doesn't contain dev/shm")
```

Comment:
A warning sounds hard. This will happen if an old checkpoint is being restored without `dev/shm` content. Maybe a Debug message would be enough.

Reply:
I've changed `Warn` to `Debug`.